web-server/v0.4/src/e2e/controllers.e2e.js

```diff
@@ -169,4 +169,43 @@ describe('controller page component', () => {
     },
     30000
   );
+
```

Comment:
Seems like we don't need to keep this code commented out.  Is there a reason to keep it?